H2Codez/Common/H2EKCommon.cpp

```diff
@@ -363,7 +363,7 @@ char __cdecl tags__fix_corrupt_fields___hook(tag_block_defintions *def, tag_bloc
 	return tags__fix_corrupt_fields_org(def, data, 1);
 }
 
-static void set_tag_data_max_size(size_t limit)
```

Comment:
but like seriously keep the static

Reply:
The variable won't change after putting the option if it's static?